views/tenants/index.erb

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <title>Tenant Info</title>
+</head>
+<body>
+  <h2>Tenant List</h2>
+
+  <% @tenants.each do |tenant| %>
+  <p class="tenant">
+    <a href="/tenants/<%= tenant.apartment_id %>"><%= tenant.name %></a>
```

Comment:
This line looks wrong to me! The id you add in using ERB in the HREF should be the tenant's *id*, not their *apartment_id*